WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -33,6 +33,14 @@ final class CollectOrderPaymentUseCase: NSObject, CollectOrderPaymentProtocol {
     ///
     private let order: Order
 
+    /// Order total in decimal number. It is lazy to avoid multiple conversions that can be costly.
```

Comment:
Creating a formatter is costly... I'm not sure how true that is of using it. More than happy with this though, since you retain the formatter anyway 😊

Reply:
That's true, I assumed it would be somehow expensive as [DateFormatter](https://sarunw.com/posts/how-expensive-is-dateformatter/), but after double-checking it I realized it is not. Thanks for the comment, I updated it in https://github.com/woocommerce/woocommerce-ios/pull/6755/commits/c5099230cea85df9d32e95a31b70819ca8ce51a8 🚀